### DISCUSSION TITLE
Running darc with Redis in Docker locally

### DISCUSSION THREAD

**Discussion by DaveyHeisen** [JarryShaw/darc#48] [Q&A] [answered]
Hi,

We are trying to run darc locally in a docker container with a local container of Redis. 
Can you please explain to us how we should set up darc, so it will connect with the local Redis container?

Is there maybe a possibility to define the REDIS_URL when starting the darc container? Or when running the darc command?

Thank you very much!

Kind regards,
Cyber Security students from Amsterdam :)

**Comment by JarryShaw** [comment, marked as answer]
Please refer to the documentation of Docker itself. There is an `-e` switch for adding environment variable to the container. You may check out the [`docker-compose.debug.yml`](https://github.com/JarryShaw/darc/blob/master/docker-compose.debug.yml) at the root folder as a sample of how to use `darc` with Docker Compose.

FYI, this may involve bridged network over the Docker VM and local machine. Appropriate solution shall depend on which OS you are using.

For more info on setup and/or deployment, please refer to the [documentation](https://darc.jarryshaw.me/en/latest/howto/deploy.html).
